core/src/smartcontracts/wasm.rs

```diff
@@ -439,8 +447,61 @@ impl<'wrld> Runtime<'wrld> {
         Ok(value_offset)
     }
 
-    /// Host defined function which prints given string. When calling
-    /// this function, module serializes ISI to linear memory and
+    fn query_max_log_level(mut caller: Caller<State>) -> Result<WasmUsize, Trap> {
+        unimplemented!()
```

Comment:
why it's unimplemented?

Reply:
sorry, I found this was missing and am working on it